portfolio/src/main/webapp/index.html

```diff
@@ -39,6 +39,9 @@
           <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#skills">Skills</a></li>
           <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#interests">Interests</a></li>
           <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#comments">Comments</a></li>
+          <br>
+          <br>
+          <li class="nav-item"><a class="nav-link js-scroll-trigger" href="" id="login">LOGIN</a></li>
```

Comment:
Not a hard-and-fast rule, but usually links to nowhere are specified with `href="#"`